assets/scripts/segments/default.js

```diff
@@ -1,33 +1,125 @@
 export const DEFAULT_SEGMENTS = {
-  false: [ // Right-hand traffic
+  false: [
+    // Right-hand traffic
     { type: 'sidewalk', variant: { 'sidewalk-density': 'dense' }, width: 6 },
     { type: 'sidewalk-tree', variant: { 'tree-type': 'big' }, width: 2 },
-    { type: 'transit-shelter', variant: { orientation: 'left', 'transit-shelter-elevation': 'street-level' }, width: 9 },
-    { type: 'sidewalk-lamp', variant: { 'lamp-orientation': 'right', 'lamp-type': 'modern' }, width: 2 },
-    { type: 'bus-lane', variant: { direction: 'inbound', 'bus-asphalt': 'shared' }, width: 12 },
-    { type: 'drive-lane', variant: { direction: 'inbound', 'car-type': 'car' }, width: 9 },
+    {
+      type: 'transit-shelter',
+      variant: {
+        orientation: 'left',
+        'transit-shelter-elevation': 'street-level'
+      },
+      width: 9
+    },
+    {
+      type: 'sidewalk-lamp',
+      variant: { 'lamp-orientation': 'right', 'lamp-type': 'modern' },
+      width: 2
+    },
+    {
+      type: 'bus-lane',
+      variant: { direction: 'inbound', 'bus-asphalt': 'shared' },
+      width: 12
+    },
+    {
+      type: 'drive-lane',
+      variant: { direction: 'inbound', 'car-type': 'car' },
+      width: 9
+    },
     { type: 'divider', variant: { 'divider-type': 'bush' }, width: 3 },
-    { type: 'turn-lane', variant: { direction: 'outbound', 'turn-lane-orientation': 'left-straight' }, width: 10 },
-    { type: 'parking-lane', variant: { 'parking-lane-direction': 'outbound', 'parking-lane-orientation': 'right' }, width: 7 },
+    {
+      type: 'turn-lane',
+      variant: {
+        direction: 'outbound',
+        'turn-lane-orientation': 'left-straight'
+      },
+      width: 10
+    },
+    {
+      type: 'parking-lane',
+      variant: {
+        'parking-lane-direction': 'outbound',
+        'parking-lane-orientation': 'right'
+      },
+      width: 7
+    },
     { type: 'divider', variant: { 'divider-type': 'planter-box' }, width: 4 },
-    { type: 'bike-lane', variant: { direction: 'outbound', 'bike-asphalt': 'green' }, width: 6 },
-    { type: 'sidewalk-lamp', variant: { 'lamp-orientation': 'left', 'lamp-type': 'modern' }, width: 2 },
+    {
+      type: 'bike-lane',
+      variant: {
+        direction: 'outbound',
+        'bike-asphalt': 'green',
+        elevation: 'elevation-lower'
```

Comment:
I think we need this to be:

```
elevation: 'road'
```

(Otherwise "new street" shows an error segment.)

Reply:
done